README.md

```diff
@@ -1,4 +1,16 @@
-# Hello-World
-#Project built with HTML and CSS
+Project Hello World
+Add 3 files (each one in a separate commit):
+A .html file that includes a header with the text "Hello Microverse!"
+A .css file that includes one class with styles for your header.
+A test.md file with the text "This file should be ignored by git".
 
-#It is included linters and gitignore
+Built With
+HTML
+CSS
+
+
+Authors
+👤 Pedro Marques
+
+GitHub: @ppmarq1
+Twitter: @ppmarq1
```

Comment:
- [ ] [OPTIONAL] Please be sure to use proper styling for the `README.md`, you can check the raw text of Microverse's readme template to do so.